Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,138 @@
+/*GLOBAL STLES*/
+
+html {
+	padding: 1%;
+	border-bottom: 15px solid #005C89;
+	border-top: 15px solid #005C89;
+}
+
+nav {
+	text-align: center;
+	font-family: Open Sans, sans-serif;
+}
+
+
+body {
+	background-color: #f6f6f6;
+
+}
+
+img {
+	width: 90%;
+	border: 2px solid #005C89;
+	padding: 3px;
+}
+
+h1 {
+	font-size: 60px;
+	font-family: Lora, Serif;
+	color: #272A32;
+}
+
+h2 {
+	font-family: Lora, Serif;
+	font-size: 30px;
+	color: #272A32;
+}
+
+h3 {
+	font-family: Lora, Serif;
+	font-size: 24px;
+	color: #272A32;
+}
+
+h4 {
+	font-family: Open Sans, sans-serif;
+	font-size: 21px;
+}
```

Comment:
We didn't mention this just yet, but I can see you're advanced enough — I would combine much of these `heading` styles into one declaration, since they share a couple of the same properties, just to keep it easier to edit in case you need to change either of those "global" heading styles:
```
h1, h2, h3 {
  font-family: 'Lora', serif;
  color: #272A32;
}

h1 {
  font-size: 60px;
}

h2 {
  font-size: 30px;
}

h3 {
  font-size: 24px;
}
```